form/form_collections.rst

```diff
@@ -305,17 +300,17 @@ you'll replace with a unique, incrementing number (e.g. ``task[tags][3][name]``)
     const addFormToCollection = (e) => {
       const collectionHolder = document.querySelector('.' + e.currentTarget.dataset.collectionHolderClass);
 
-      const item = document.createElement('li');
+      const element = document.createElement('li');
```

Comment:
Why not call the variable `listItem` everywhere instead?

```suggestion
      const listItem = document.createElement('li');

Reply:
Short answer: Cause it might also be a `<div>` (or whatever).

Long answer: I really like how @Oipnet in https://github.com/symfony/symfony-docs/pull/15455 added the `data-index` attribute to Twig to get it out of JavaScript. So the *only* implementation-specific thing that's left in JavaScript is the HTML element to be added. I even thought about adding a `data-element` attribute for that... In the end, I just changed the variable name to something *very* generic :-)